### ISSUE TITLE
Rails-3.2.1 and write_inheritable_attribute/read_inheritable_attribute

### ISSUE THREAD

**Issue by greendog** [albertosaurus/power_enum#7]
Hey.
I am using enumerations_mixin for rails-3.2.1 and get the following error:

``` ruby
undefined method `read_inheritable_attribute'  for #<Class:0xe14c6bc>
```

``` ruby
undefined method ` write_inheritable_attribute'  for #<Class:0xe14c6bc>
```

http://apidock.com/rails/Class/read_inheritable_attribute
for rails-3.2.1  this is method not available 
it should be replaced by 

``` ruby
class_attribute
```

**Comment by albertosaurus** [comment]
Thanks, I'll try to look into it over the weekend.

**Comment by albertosaurus** [comment]
Fixed in version 0.6.1.
